instrument/Facilities.xml

```diff
@@ -90,6 +91,7 @@
   </instrument>
 
   <instrument name="GEM">
+    <zeropadding size="8" startRunNumber="97480"/>
```

Comment:
I checked on the archive and zero padding starts with the previous file
```suggestion
    <zeropadding size="8" startRunNumber="97479"/>
```

Reply:
Sorry used the run on the issue - should have checked!